client/src/app/(profile)/profile/page.tsx

```
@@ -170,21 +189,37 @@ export default function ProfilePage() {
-                    <Link
-                      href={item.link}
-                      className="flex flex-col items-center text-center gap-2"
-                    >
-                      <item.icon className="w-6 h-6 text-primary" />
-                      <h3 className="font-medium">{item.label}</h3>
-                      <p className="text-xs text-muted-foreground">
-                        {item.desc}
-                      </p>
-                    </Link>
+                    {item.label === "Logout" ? (
+                      <div onClick={handleLogout} className="flex flex-col items-center text-center gap-2">
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)